app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        if valid_type_param
+          render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
```

Comment:
a esto solemos llamarlo directamente con el nombre del recurso en plural 
```suggestion
          render json: notes, status: :ok, each_serializer: IndexNoteSerializer
```